src/Styles/icon.scss

```diff
@@ -29,356 +29,359 @@
+    content: "\e936";
   }
   &.redo:before {
-    content: '\e959';
+    content: "\e959";
   }
   &.refresh:before {
-    content: '\e949';
+    content: "\e949";
   }
   &.remove:before {
-    content: '\e90d';
+    content: "\e90d";
   }
   &.search:before {
-    content: '\e916';
+    content: "\e916";
   }
   &.server2:before {
-    content: '\e94c';
+    content: "\e94c";
   }
   &.shuffle:before {
-    content: '\e94e';
+    content: "\e94e";
   }
   &.square:before {
-    content: '\e93e';
+    content: "\e93e";
   }
   &.square-check:before {
-    content: '\e943';
+    content: "\e943";
   }
   &.star:before {
-    content: '\e930';
+    content: "\e930";
   }
   &.stop:before {
-    content: '\e931';
+    content: "\e931";
   }
   &.text-block:before {
-    content: '\e911';
+    content: "\e911";
   }
   &.trash:before {
-    content: '\e915';
+    content: "\e915";
   }
   &.triangle:before {
-    content: '\e92c';
+    content: "\e92c";
   }
   &.undo:before {
-    content: '\e95a';
+    content: "\e95a";
   }
   &.user:before {
-    content: '\e919';
+    content: "\e919";
   }
   &.user-add:before {
-    content: '\e91e';
+    content: "\e91e";
   }
   &.user-outline:before {
-    content: '\e932';
+    content: "\e932";
   }
   &.user-remove:before {
-    content: '\e939';
+    content: "\e939";
   }
   &.users-trio:before {
-    content: '\e945';
+    content: "\e945";
   }
   &.wand:before {
-    content: '\e94d';
+    content: "\e94d";
   }
   &.wood:before {
-    content: '\e970';
+    content: "\e970";
   }
   &.wrench:before {
-    content: '\e971';
+    content: "\e971";
   }
   &.zap:before {
-    content: '\e973';
+    content: "\e973";
   }
   &.bell-ringing:before {
-    content: '\e935';
+    content: "\e935";
   }
   &.calendar:before {
-    content: '\e938';
+    content: "\e938";
   }
   &.cloud:before {
-    content: '\e93f';
+    content: "\e93f";
   }
   &.cubo-nav:before {
-    content: '\e91a';
+    content: "\e91a";
   }
   &.dashboards:before {
-    content: '\e93a';
+    content: "\e93a";
   }
   &.disks-nav:before {
-    content: '\e946';
+    content: "\e946";
   }
   &.graphline-2:before {
-    content: '\e924';
+    content: "\e924";
   }
   &.users-duo:before {
-    content: '\e93b';
+    content: "\e93b";
   }
   &.wrench-nav:before {
-    content: '\e972';
+    content: "\e972";
   }
   &.book:before {
-    content: '\e966';
+    content: "\e966";
   }
   &.book-code:before {
-    content: '\e967';
+    content: "\e967";
   }
   &.book-pencil:before {
-    content: '\e965';
+    content: "\e965";
   }
   &.bucket-solid:before {
-    content: '\e962';
+    content: "\e962";
   }
   &.funnel-outline:before {
-    content: '\e963';
+    content: "\e963";
   }
   &.funnel-solid:before {
-    content: '\e964';
+    content: "\e964";
   }
   &.layers:before {
-    content: '\e96c';
+    content: "\e96c";
   }
   &.share:before {
-    content: '\e96d';
+    content: "\e96d";
   }
-}
+}  
```

Comment:
did this file get modified automatically? i think our standards in other codebases are to use single quotes `''`.

the following text is copied from another comment:

>text files need newlines at the end of them, see https://stackoverflow.com/a/729795/92446.

>Here's how to automatically add them when using VSCode:
https://newbedev.com/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
Ahh yes these files are autogenerated by a 3rd party tool that does Icon generation. I'll fix the code convention stuff. Good catch !!